diff --git a/solenv/gdb/libreoffice/cppu.py b/solenv/gdb/libreoffice/cppu.py
index 1e3b25e..72fd263 100644
--- a/solenv/gdb/libreoffice/cppu.py
+++ b/solenv/gdb/libreoffice/cppu.py
@@ -71,6 +71,17 @@ class UnoReferencePrinter(object):
     def _itype(self):
         return self.value.type.template_argument(0).pointer()
 
+class BeansXPropertySetPrinter(object):
+    def __init__(self, typename, value):
+        self.value = value
+        self.typename = typename.replace('com::sun::star::', '')
+
+    def to_string(self):
+        impl = self.value.cast(self.value.dynamic_type)
+        xInfo = impl['getPropertySetInfo']
+        xInfo(self.value)
+        return "debug, foo"
+
 class UnoSequencePrinter(object):
     '''Prints UNO Sequence'''
 
@@ -152,6 +163,7 @@ def build_pretty_printers():
 
     # basic UNO stuff
     printer.add('_uno_Any', UnoAnyPrinter)
+    printer.add('com::sun::star::beans::XPropertySet', BeansXPropertySetPrinter)
     printer.add('com::sun::star::uno::Any', UnoAnyPrinter)
     printer.add('com::sun::star::uno::Reference', UnoReferencePrinter)
     printer.add('com::sun::star::uno::Sequence', UnoSequencePrinter)
